### PR TITLE
Multi arch for nonroot image

### PR DESCRIPTION
**Pull request by mkumatag**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change

/kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
This PR is for pushing nonroot for non-x86 platforms and push fat manifest as well, because of this nonroot tests are failing on non-x86 platforms.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

File: test/images/nonroot/BASEIMAGE [hunk 1]
```diff
@@ -0,0 +1,4 @@
+amd64=k8s.gcr.io/debian-base-amd64:v1.0.0
+arm=k8s.gcr.io/debian-base-arm:v1.0.0
+arm64=k8s.gcr.io/debian-base-arm64:v1.0.0
+ppc64le=k8s.gcr.io/debian-base-ppc64le:v1.0.0
```
Comment (dims) [inline test/images/nonroot/BASEIMAGE]: s390x ?

Comment (mkumatag) [reply test/images/nonroot/BASEIMAGE]: repushed adding s390x as well.

**Comment by dims** [comment]
let's bump the test/images/nonroot/VERSION too. So we can distinguish between the current failing image and the newly built image that hopefully works @mkumatag

**Comment by mkumatag** [comment]
> let's bump the test/images/nonroot/VERSION too. So we can distinguish between the current failing image and the newly built image that hopefully works @mkumatag

Well, the current image fails with just `exec format` error which is just an unsupported image format. I don't see any changes in the code as well.

One more advantage of retaining this image name is that - previous release tests can also this version.

**Comment by dims** [comment]
ack/agree @mkumatag 

/approve
/lgtm

**Comment by mkumatag** [comment]
@listx can you please help me pushing this image?

**Comment by listx** [comment]
Sorry for the wait; I just pushed it.

Here's the log:

```
[火 14 21:21:00] a25c6aa3506:----:master● linusa_enif ~/go/src/k8s.io/kubernetes/test/images
 $ make all-push WHAT=nonroot
./image-util.sh build nonroot
Building image for nonroot ARCH: amd64...
~/go/src/k8s.io/kubernetes/_tmp/test-images-build.Fu6ui2 ~/go/src/k8s.io/kubernetes/test/images
Sending build context to Docker daemon  5.632kB
Step 1/2 : FROM k8s.gcr.io/debian-base-amd64:v1.0.0
v1.0.0: Pulling from debian-base-amd64
39fafc05754f: Pull complete 
Digest: sha256:5f25d97ece9076612b64bb551e12f1e39a520176b684e2d663ce1bd53c5d0618
Status: Downloaded newer image for k8s.gcr.io/debian-base-amd64:v1.0.0
 ---> 204e96332c91
Step 2/2 : USER 1234
 ---> Running in 38a671383ed6
Removing intermediate container 38a671383ed6
 ---> c49c7331e9dc
Successfully built c49c7331e9dc
Successfully tagged gcr.io/kubernetes-e2e-test-images/nonroot-amd64:1.0
~/go/src/k8s.io/kubernetes/test/images
Building image for nonroot ARCH: arm...
~/go/src/k8s.io/kubernetes/_tmp/test-images-build.A0dnzi ~/go/src/k8s.io/kubernetes/test/images
Sending build context to Docker daemon  5.632kB
Step 1/2 : FROM k8s.gcr.io/debian-base-arm:v1.0.0
v1.0.0: Pulling from debian-base-arm
1b0dd6356894: Pull complete 
Digest: sha256:e7e5947cfa8f234fdbdb06fb31c4c7ec00d856c1c7967066510bf022081fc34d
Status: Downloaded newer image for k8s.gcr.io/debian-base-arm:v1.0.0
 ---> 468d130e32a6
Step 2/2 : USER 1234
 ---> Running in daca921359ff
Removing intermediate container daca921359ff
 ---> 42802478b4ae
Successfully built 42802478b4ae
Successfully tagged gcr.io/kubernetes-e2e-test-images/nonroot-arm:1.0
~/go/src/k8s.io/kubernetes/test/images
Building image for nonroot ARCH: arm64...
~/go/src/k8s.io/kubernetes/_tmp/test-images-build.8uOPgO ~/go/src/k8s.io/kubernetes/test/images
Sending build context to Docker daemon  5.632kB
Step 1/2 : FROM k8s.gcr.io/debian-base-arm64:v1.0.0
v1.0.0: Pulling from debian-base-arm64
9f9ba9541db2: Pull complete 
Digest: sha256:17be039c7035bd0897d954c51914ad41cd7e2b0b7c170b3d89ed021833df2fb1
Status: Downloaded newer image for k8s.gcr.io/debian-base-arm64:v1.0.0
 ---> 4f7e1a9aba5b
Step 2/2 : USER 1234
 ---> Running in 199940992d97
Removing intermediate container 199940992d97
 ---> 1a5e849dbe78
Successfully built 1a5e849dbe78
Successfully tagged gcr.io/kubernetes-e2e-test-images/nonroot-arm64:1.0
~/go/src/k8s.io/kubernetes/test/images
Building image for nonroot ARCH: ppc64le...
~/go/src/k8s.io/kubernetes/_tmp/test-images-build.FZk3Yp ~/go/src/k8s.io/kubernetes/test/images
Sending build context to Docker daemon  5.632kB
Step 1/2 : FROM k8s.gcr.io/debian-base-ppc64le:v1.0.0
v1.0.0: Pulling from debian-base-ppc64le
2f8fd71a7e15: Pull complete 
Digest: sha256:60371f4bd36c4803695dc83e19efd37d4fefd479b369b0a9aa1c12e3c3d17f8e
Status: Downloaded newer image for k8s.gcr.io/debian-base-ppc64le:v1.0.0
 ---> c287b270ebbb
Step 2/2 : USER 1234
 ---> Running in 59ada6186479
Removing intermediate container 59ada6186479
 ---> 3f95abcc9893
Successfully built 3f95abcc9893
Successfully tagged gcr.io/kubernetes-e2e-test-images/nonroot-ppc64le:1.0
~/go/src/k8s.io/kubernetes/test/images
Building image for nonroot ARCH: s390x...
~/go/src/k8s.io/kubernetes/_tmp/test-images-build.VSeWW2 ~/go/src/k8s.io/kubernetes/test/images
Sending build context to Docker daemon  5.632kB
Step 1/2 : FROM k8s.gcr.io/debian-base-s390x:v1.0.0
v1.0.0: Pulling from debian-base-s390x
cf0d51cc3343: Pull complete 
Digest: sha256:55309d4261979b7660de562f9d4b16de9576df860e8f466f29251f2e94bd85fd
Status: Downloaded newer image for k8s.gcr.io/debian-base-s390x:v1.0.0
 ---> 513e660cbbbc
Step 2/2 : USER 1234
 ---> Running in ab601629d85a
Removing intermediate container ab601629d85a
 ---> fdfd9b4d2cc5
Successfully built fdfd9b4d2cc5
Successfully tagged gcr.io/kubernetes-e2e-test-images/nonroot-s390x:1.0
~/go/src/k8s.io/kubernetes/test/images
./image-util.sh push nonroot
The push refers to repository [gcr.io/kubernetes-e2e-test-images/nonroot-amd64]
fe9a8b4f1dcc: Layer already exists 
1.0: digest: sha256:8b63951ad4dfc93d55863725e063e49a191f704b7bcf68231639a47288153c78 size: 529
The push refers to repository [gcr.io/kubernetes-e2e-test-images/nonroot-arm]
287e0147e8b0: Pushed 
1.0: digest: sha256:2c078301e60d77378517e338d8e9f3ef4b9614a31eddd3fc37d56751d8b057ca size: 529
The push refers to repository [gcr.io/kubernetes-e2e-test-images/nonroot-arm64]
186143d216c2: Pushed 
1.0: digest: sha256:602c129ca0ccc5efd74ea6aeb7ff3e94d5415ebd5157f9e186a04baa6607243f size: 529
The push refers to repository [gcr.io/kubernetes-e2e-test-images/nonroot-ppc64le]
f05f5a927c6b: Pushed 
1.0: digest: sha256:1fbcd1d44d5b9971adefa7afb54c19bd068feb96337b296db2c092f07e2a6580 size: 529
The push refers to repository [gcr.io/kubernetes-e2e-test-images/nonroot-s390x]
fbd26cd2461d: Pushed 
1.0: digest: sha256:5c123427702a7b3e7721aa6464b260e8e7dfe5304be8f202607a23fbeb4fdd5e size: 529
Created manifest list gcr.io/kubernetes-e2e-test-images/nonroot:1.0
Pushed ref gcr.io/kubernetes-e2e-test-images/nonroot@sha256:8b63951ad4dfc93d55863725e063e49a191f704b7bcf68231639a47288153c78 with digest: sha256:8b63951ad4dfc93d55863725e063e49a191f704b7bcf68231639a47288153c78
Pushed ref gcr.io/kubernetes-e2e-test-images/nonroot@sha256:2c078301e60d77378517e338d8e9f3ef4b9614a31eddd3fc37d56751d8b057ca with digest: sha256:2c078301e60d77378517e338d8e9f3ef4b9614a31eddd3fc37d56751d8b057ca
Pushed ref gcr.io/kubernetes-e2e-test-images/nonroot@sha256:602c129ca0ccc5efd74ea6aeb7ff3e94d5415ebd5157f9e186a04baa6607243f with digest: sha256:602c129ca0ccc5efd74ea6aeb7ff3e94d5415ebd5157f9e186a04baa6607243f
Pushed ref gcr.io/kubernetes-e2e-test-images/nonroot@sha256:1fbcd1d44d5b9971adefa7afb54c19bd068feb96337b296db2c092f07e2a6580 with digest: sha256:1fbcd1d44d5b9971adefa7afb54c19bd068feb96337b296db2c092f07e2a6580
Pushed ref gcr.io/kubernetes-e2e-test-images/nonroot@sha256:5c123427702a7b3e7721aa6464b260e8e7dfe5304be8f202607a23fbeb4fdd5e with digest: sha256:5c123427702a7b3e7721aa6464b260e8e7dfe5304be8f202607a23fbeb4fdd5e
sha256:4bd7ae247de5c988700233c5a4b55e804ffe90f8c66ae64853f1dae37b847213
```

**Comment by mkumatag** [comment]
> Sorry for the wait; I just pushed it.
> 
> Here's the log:

Thanks a lot @listx , you made my day 👍

**Pull request merged**